### PR TITLE
Fix callback

### PR DESCRIPTION
**Pull request by Luni-4**
- Fix `ffms2` callback on `Windows`
- Move callback functions out the `ffms2` python lib

### CODE REVIEW

File: ffms2/libffms2.py [hunk 1]
```diff
@@ -177,12 +175,17 @@ class FFMS_AudioProperties(Structure):
...
     ("FirstTime", c_double),
     ("LastTime", c_double),
 ]
-TIndexCallback = FUNCTYPE(c_int, int64_t, int64_t, c_void_p)
+
+# Since the ffms2.dll is cross-compiled, the calling convention is cdecl.
+# If you have compiled the ffms2.dll directly on Windows using MSVC,
+# you will probably use the stdcall convention, in that case,
+# replace CFUNCTYPE with WINFUNCTYPE.
+TIndexCallback = CFUNCTYPE(c_int, c_int64, c_int64, c_void_p)
```
Comment (rr-) [inline ffms2/libffms2.py]: Is there a way to detect this ootb?

File: ffms2/__main__.py [hunk 2]
```diff
@@ -131,22 +146,22 @@ def main():
...
         if args.timecodes:
-            stdout_write("Writing timecodes…\n")
+            stdout_write("Writing timecodes...\n")
             for track in index.tracks:
                 if track.type == ffms2.FFMS_TYPE_VIDEO:
                     track.write_timecodes()
 
         if args.keyframes:
-            stdout_write("Writing keyframes…\n")
+            stdout_write("Writing keyframes...\n")
```
Comment (rr-) [inline ffms2/__main__.py]: What's wrong with "…"? On one hand you remove the Python 2 trick to deal with non-utf8 terminal encoding, on another you coerce output into ANSI

Comment (Luni-4) [reply ffms2/__main__.py]: The Github Actions' terminal emulator couldn't read that symbol in both ways, don't know why.

File: ffms2/libffms2.py (hunk 1 above)
Comment (Luni-4) [reply ffms2/libffms2.py]: It might be a way, let me check, thanks for the advice!

Comment (Luni-4) [reply ffms2/libffms2.py]: Fixed

File: ffms2/__main__.py (hunk 2 above)
Comment (Luni-4) [reply ffms2/__main__.py]: I'm going to iterate over this one in the next PR

**Pull request merged**